wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/de.md

```diff
@@ -0,0 +1,66 @@
+# Befehle für das Turnier-Management
+
+Die folgenden Chatbefehle sind für die Fernverwaltung von Mehrspieler-Turnierräumen vorgesehen:
+
+- `!mp make <name>` - Erstellt einen Turnierraum mit dem angegebenen Namen. Maximal 4 solcher Räume können erstellt werden.
+  - Dieser Raum ist insofern besonders, da er nicht geschlossen wird, sobald alle Spieler den Raum verlassen haben. Außerdem ist er vor Spielern, die ihm selbst beitreten wollen, passwortgeschützt.
+  - Wenn der Raum nicht mehr weiter genutzt werden soll, benutzte `!mp close`, um den Raum zu schließen.
+- `!mp makeprivate <name>` - Erstellt einen privaten Turnierraum mit dem angegebenen Namen. Dieser Befehl funktioniert wie `!mp make`, aber der Spielverlauf ist nur für den Raumersteller und die Teilnehmer sichtbar.
+- `!mp name <title>` - Ändert den Raumnamen.
+- `!mp invite <username>` - Lädt einen Spieler in den Raum ein.
+  - Bedenke, dass dies *keine* Blockierungen privater Nachrichten umgeht, die im osu!-Client einstellbar sind. D. h., dass dein Turnierpersonal den Spielern mitteilen muss, "Private Nachrichten nur von Freunden" in den osu!-Einstellungen zu deaktivieren.
+- `!mp lock` - Sperrt den Raum, sodass Spieler nicht mehr ihr Team und ihren Slot ändern können.
+- `!mp unlock` - Kehrt die obige Aktion um.
+- `!mp size <size>` - Stellt die Menge an verfügbaren Slots (1-16) im Raum ein.
+- `!mp set <teammode> [<scoremode>] [<size>]` - Stellt verschiedene Raumeigenschaften ein.
+  - `teammode` - 0: Head To Head, 1: Tag Coop, 2: Team Vs, 3: Tag Team Vs
+  - `scoremode` - 0: Score, 1: Accuracy, 2: Combo, 3: Score V2
+- `!mp move <username> <slot>` - Verschiebt einen Spieler innerhalb des Raums zum angegebenen Slot (Startindex 1).
+- `!mp host <username>` - Transferiert die Host-Rechte zu diesem Spieler.
+- `!mp clearhost` - Löscht die Host-Rechte des Raums.
```

Comment:
I used `Host-Rechte` in the two lines above to make it clear that not the player itself gets kicked out. What I'm unsure about is if `clearhost` actually clears just the rights of the current host, so I would like some feedback.

Reply:
What this command does is remove host rights from the current host (provided there is one), so no one in the room will be the host. You can test this yourself by creating a room in the client and running the command. Not sure what other effects you are expecting. Keep in mind that "Host" and "Room Creator/Referee" are different roles, and the latter don't necessarily have to be in the room.

Reply:
Yeah, this is exactly what I meant. The player who is host can't execute host commands anymore, but stays in the room.

Reply:
Oh if all you're asking for is feedback on the wording, then yeah this works 👍